src/Ogooreck.Sample.BusinessLogic.FSharp.Tests/Deciders/BankAccountPrimitives.fs

```diff
@@ -0,0 +1,39 @@
+﻿module Deciders.BankAccountPrimitives
+
+open System
+
+[<Struct>]
+type AccountId = AccountId of Guid
```

Comment:
Consider using FSharp.UMX so you don't need magic stuff to happen for STJ or Json.NET to work. 
(Not any less safe than using a struct in terms of guaranteeing no bad values)

Reply:
Updated in https://github.com/oskardudycz/Ogooreck/pull/13/commits/db717592801bbff360e02f0b7e5bd6a3e5d9c1a0.

Reply:
Lovely now!